pallets/reversible-transfers/src/lib.rs

```diff
@@ -853,62 +839,60 @@ pub mod pallet {
 			// For native balances, transfer held funds to treasury (fee) and interceptor
 			// (remaining)
 			if let Ok((call, _)) = T::Preimages::peek::<RuntimeCallOf<T>>(&pending.call) {
-				if let Ok(assets_call) = call.clone().try_into() {
-					if let pallet_assets::Call::transfer_keep_alive { id, .. } = assets_call {
```

Comment:
Are these just cosmetic changes? Costs much more to review because everything changed...